src/DevTools/DevTools.php

```diff
@@ -59,7 +59,7 @@ public function onCommand(CommandSender $sender, Command $command, string $label
 					$plugins = $this->getServer()->getPluginManager()->getPlugins();
 					$succeeded = $failed = [];
 					foreach($plugins as $plugin){
-						if($this->makePluginCommand($sender, $command, $label, [$plugin->getName()])){
+						if(($plugin->getPluginLoader() instanceof FolderPluginLoader) && $this->makePluginCommand($sender, $command, $label, [$plugin->getName()])){
```

Comment:
this will still yield the "Failed to build", I meant it should just skip non-folder-plugins entirely.